Dev

dmbavykin opened the pull request.

Commit:
configs

Commit:
models added

Commit:
cart added

Commit:
cart tests added

Commit:
checkout added

Commit:
orders panel added

Commit:
rubocop ref

Commit:
changing current_user logic

kirillshevch commented on app/controllers/shoppy_cartus/orders_controller.rb:
```diff
@@ -0,0 +1,29 @@
+      ShoppyCartus::Order.aasm_states.include?(params[:state]&.to_sym)
+    end
+  end
+end
```
you forgot about authorization

kirillshevch replied:
show/confirm

kirillshevch commented:
Please add CircleCI

kirillshevch commented:
You don't need this folder https://github.com/6aBblKuH/shoppy_cartus/tree/dev/test

Commit:
fix abilities

	modified:   Gemfile.lock
	modified:   app/controllers/shoppy_cartus/orders_controller.rb
	modified:   app/models/shoppy_cartus/ability.rb
	modified:   lib/shoppy_cartus/version.rb
	deleted:    test/integration/navigation_test.rb
	deleted:    test/shoppy_cartus_test.rb
	deleted:    test/test_helper.rb
